### DISCUSSION TITLE
Incoming streams experience rapid disconnects and reconnects, using rtmp push

### DISCUSSION THREAD

**Discussion by hernanrz** [OvenMediaLabs/OvenMediaEngine#1413] [Help]
We're receiving over 30 streams, each with multiple independent viewers, each stream is re-streamed to a nginx-rtmp instance using the startPush API method

After a few hours of running the server streams started to disconnect and reconnect rapidly and we found this in the logs, if there's anything we can tune in our settings please let us know, thanks!

![image](https://github.com/AirenSoft/OvenMediaEngine/assets/13426225/b030bd3f-33ea-445d-92af-f2b162bb0802)
[log.txt](https://github.com/AirenSoft/OvenMediaEngine/files/12876359/message.txt)

My understanding is that these kind of "queue... has exceeded the threshold" messages means we're not processing streams fast enough, or we don't have enough workers? Does this mean we need more workers to handle the amount of rtmp pushes we're doing? 

Also, is (1) for OVT worker count enough for the amount of streams we are ingesting?

Origin config: https://gist.github.com/hernanrz/40243c766094e5ffd374397dcfc2a57e

**Comment by getroot** [comment]
Your configuration doesn't involve video encoding, and with just OVT publisher and RTMPPush publisher, OvenMediaEngine should typically handle around 30 streams without any issues. Can you provide information about the CPU usage at the time when you observed this problem? Since OPUS encoding is in use, if the CPU core count is too low, it might have led to performance issues.

If you're still experiencing this problem even when the CPU isn't fully utilized, there's a possibility of a software bug. To identify the specific cause of the issue, please share the version of OvenMediaEngine and the complete logs, and we will further analyze the situation.

**Reply by hernanrz** [reply]
![image](https://github.com/AirenSoft/OvenMediaEngine/assets/13426225/6608c7be-c2f1-42e9-8d1d-3ab241327c8d)

CPU wasn't fully utilized by either edge or origin containers at the time of the issues, I don't have access to the logs from the time of the problem anymore, sorry

**Reply by getroot** [reply]
Does this issue no longer recur?
If you are able to reproduce it, please provide additional information so we can analyze it further.

**Reply by hernanrz** [reply]
closing as we haven't experienced this any further
